log15_test.go

```diff
@@ -284,8 +286,11 @@ func TestNetHandler(t *testing.T) {
 	select {
 	case <-time.After(time.Second):
 		t.Fatalf("Test timed out!")
-	case <-errs:
-		// ok
+	case err := <-errs:
+		if err == nil {
+			return
+		}
+		t.Fatal(err)
```

Comment:
I think the conditional should be inverted to `err != nil` and `t.Fatal(err)` moved inside the `if` to match typical Go error handling patterns. It also removes the need for the explicit return statement.